### PR TITLE
Update doc/read-the-docs-site/README.md

### PR DESCRIPTION
**Pull request by zeme-iohk**

### CODE REVIEW

File: doc/read-the-docs-site/README.md [hunk 1]
```diff
@@ -1,26 +1,17 @@
 # Plutus and Marlowe documentation site
 
 This is a sphinx site.
-Assuming you're in a `nix develop` shell, from the top-level, you can either build the site directly:
 
-```
-sphinx-build -j 4 -n doc/read-the-docs-site doc/read-the-docs-site/_build
-```
+Run `nix develop` to enter a development shell with `sphinx-build` and `sphinx-auotbuild`.
```
Comment (michaelpj) [inline doc/read-the-docs-site/README.md:5]:
```suggestion
Run `nix develop` to enter a development shell with `sphinx-build` and `sphinx-autobuild`.
```

File: nix/cells/plutus/library/make-plutus-shell.nix [hunk 2]
```diff
@@ -92,17 +92,17 @@ inputs.std.lib.dev.mkShell {
     {
       package = cell.packages.sphinx-build-readthedocs-site;
       category = "docs";
-      help = "Build the docs locally with output in doc/_build";
+      help = "Build the docs locally in doc/_build";
```
Comment (michaelpj) [inline nix/cells/plutus/library/make-plutus-shell.nix:95]: folder is wrong?

Comment (zeme-iohk) [reply nix/cells/plutus/library/make-plutus-shell.nix]: yes, corrected

File: doc/read-the-docs-site/README.md (hunk 1 above)
Comment (zeme-iohk) [reply doc/read-the-docs-site/README.md]: 👍

**Pull request merged**